style.css

```diff
@@ -74,6 +73,59 @@
   position: absolute;
 }
 
+.shadow {
+   box-shadow: 0 -1px 20px 0px rgba(0, 0, 0, 0.3);
```

Comment:
when the value is 0 there's no need to add the `px`, I would move this directly to the footer element